ditk/doc/annotated/README.md

```diff
@@ -0,0 +1,65 @@
+# Annonated "Algorithm-Code" Documation
+
+Python Annonated Documation Generation Script
+
+# Demo
+
+<div align="center">
+    <a href="https://github.com/opendilab/DI-toolkit"><img width="800px" height="auto" src="demo.png"></a>
+</div>
+
+# Usage
```

Comment:
maybe we should also add the three-step statements in the original README, which is beneficial to explain the purpose of this module more detailedly.